src/lang/lwc_pt.properties

```diff
@@ -0,0 +1,635 @@
+Tipo:%yellow% %type% \n\
+\n\
+Proteção:%yellow% %protection% \n\
+Criado por:%yellow% %creator%
+lwc.history.details.destroyer=Removido por:%yellow% %player%
+lwc.history.details.econcharge=Cobrança de economia:%dark_red% %charge% %currencyname% \n\
+Desconto?: %discount%
+lwc.history.details.dates=\n\
+%dark_red%Datas \n\
+Criado em:%yellow% %date%
+lwc.history.details.destroyed=%dark_red%Removido em: %date%
+lwc.history.details.timeago=%yellow%%time% atrás
+
+lwc.remove.allprotections=Você está prestes a%dark_red% REMOVER PERMANENTEMENTE%white% todas as proteções que possui. \n\
+Você tem%dark_red% certeza absoluta?%white% Por favor digite%yellow% /lwc confirm%white% para confirmar que quer fazer isso;
+lwc.remove.allprotections.success=%dark_green%Todas as suas proteções foram removidas.
+
+lwc.debug.standard=%dark_green%Permissões padrões:
+lwc.debug.mod=%yellow%Permissões de moderador:
+lwc.debug.admin=%dark_red%Permissões de administrador:
+
+lwc.flags.available=Flags disponíveis: %flags%
+
+lwc.devmode.warning=Você está prestes a%dark_red%habilitar o Modo Desenvolvedor%white% para%dark_red% %player% \n\
+%white%O Modo Desenvolvedor dará a eles%dark_red% controle absoluto sobre o LWC até que desloguem%white% ou%yellow% /lwc dev disable %player%%white%é utilizado. \n\
+Por favor confirme que quer fazer isso usando o comando \n\
+%yellow%/lwc confirm
+lwc.devmode.success=%dark_green%Modo Desenvolvedor habilitado com sucesso para %player%
+lwc.devmode.disabled=%dark_green%Modo Desenvolvedor removido de %dark_red% %player%
+lwc.devmode.received=%dark_green%Você recebeu Modo Desenvolvedor.
+lwc.devmode.lost=%dark_green%Lhe foi retirado Modo Desenvolvedor.
+lwc.devmode.nodevmode=%dark_red%O jogador não tem Modo Desenvolvedor habilitado.
+lwc.devmode.permissionsmode=%dark_green%Modo de permissão alterado para:%yellow% %mode%
+
+lwc.easynotify.redstone=%dark_red%Note:%white% Redstone está atualmente permitida nessa proteção. Para desabilitar o uso de redstone nela, use%dark_green% /credstone on
+
+lwc.fix.fixed=%dark_green%%block% consertado!
+lwc.fix.clickblock=%dark_green%Clique em um bloco para começar o feitiço.
+
+lwc.admin.caches.cleared=%dark_green%Caches limpos.
+lwc.admin.dump.fileexists=%dark_red%O arquivo %file% já existe. Por favor apague ou mova este arquivo!
+lwc.admin.dump.filenotfound=%dark_red%Falha ao encontrar o idioma dentro do arquivo jar.
+lwc.admin.dump.success=%dark_green%Arquivo de idioma colocado em: %file%
+
+# Donation protection type
+donation=Doação
+
+protection.typechanged=%dark_green%A proteção foi alterada para %type%
+
+###########
+## 2.1.x ##
+###########
+
+lwc.factions.blocked=%dark_red%You are not allowed to protect blocks using LWC inside of this faction!
+
+protection.modes.nolock.finalize=%dark_green%Registration of new protections is now disabled
+protection.modes.nolock.off=%dark_green%Registration of new protections is now enabled
+
+###########
+## 2.2.x ##
+###########
+
+display=Display
```

Comment:
Looks like these ones aren't translated. The most important one being `display` since that is the translation for one of the new protection types in LWC.

Reply:
Oops, totally missed these lines. I've translated these now. Thanks for the review.

Just to make sure `Monstruário` is the right word for it, the new `display` mode makes an inventory view-only, right?

Reply:
About the rename to `lwc_pt_BR.properties`:
My server doesn't seem to load the correct locale when `locale: pt_BR`, falling back to the default. The same locale file renamed to `lwc_pt.properties` (and `locale: pt`) works, though.

Reply:
Yes, display is for displaying items only. It works exactly the same as donation, however you cannot place items into it without being added to the protection.

And interesting... I think that might be a flaw in the locale system of LWC. Just leave it as `pt` for now, and if another Portuguese translation is added later I'll figure it out then. Just make sure to leave the comment at the top of the file that this is for Brazillian.

Reply:
I'll leave it as `pt` then. Thanks for the help!

Reply:
Yeah no problem, it isn't the goal to make things difficult for you. 😛 Translating is already a difficult task.